src/demo-app/button/button-demo.html

```diff
@@ -3,21 +3,21 @@
     <button md-button>flat</button>
     <button md-raised-button>raised</button>
     <button md-fab>
-      <md-icon class="md-24">check</md-icon>
+      <md-icon class="mat-24">check</md-icon>
```

Comment:
The `md-24` class actually isn't defined by us; it comes from the material design icon font, so that one needs to change back